crates/wasmtime/src/runtime/vm.rs

```diff
@@ -336,6 +336,216 @@ impl ModuleRuntimeInfo {
     }
 }
 
+/// A number of bytes that's guaranteed to be aligned to the host page size.
+///
+/// This is used to manage page-aligned memory allocations.
+#[derive(Clone, Copy, Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
+pub struct HostAlignedByteCount(
```

Comment:
I first named this `AlignedByteCount`, but then realized that the VM page size might not be the same as the host page size, so named it `HostAlignedByteCount` instead. Happy to rename this to something else if it makes sense.

Reply:
Same seems reasonable to me 👍